src/object-types.js

```diff
@@ -0,0 +1,86 @@
+// Enumeration of spawned object content origins. URL means the content is
+// fetched from a URL, FILE means it was an uploaded/dropped file, and
+// CLIPBOARD means it was raw data pasted in from the clipboard, SPAWNER means
```

Comment:
Hmm so ther eis no way to tell clipboard urls vs add media dialog url usage?

Reply:
No, looks like they are conflated in the telemetry. I assume we care more about measuring activity with this telemetry, vs UX anyway. Also, pasting a URL directly is kind of a shortcut to using the dialog. I'm going to leave it as it is.